pkg/goods/support/host-support-bundle.tmpl.yaml

```diff
@@ -448,3 +596,47 @@ spec:
         - pass:
             when: "false"
             message: "No signs of hostname changes found"
+  - textAnalyze:
+      checkName: Check if localhost resolves to 127.0.0.1
+      fileName: host-collectors/run-host/localhost-ips.txt
+      regex: 'localhost has address 127.0.0.1'
+      outcomes:
+        - fail:
+            when: "false"
+            message: "'localhost' does not resolve to 127.0.0.1 ip address.  Ensure your /etc/hosts file contains an entry for 'localhost' with a loopback address of 127.0.0.1"
+        - pass:
+            when: "true"
+            message: "'localhost' resolves to 127.0.0.1 ip address"
+  - textAnalyze:
+      checkName: Check if SELinux is enabled
+      fileName: host-collectors/run-host/sestatus.txt
+      regex: '(?m)^Current mode:\s+enforcing'
+      ignoreIfNoFiles: true
+      outcomes:
+        - fail:
+            when: "true"
+            message: "SELinux enforcement is not supported, disable it or set to permissive mode to proceed"
+        - pass:
+            when: "false"
+            message: "SELinux is disabled as expected"
```

Comment:
I think we tested and an install worked with SELinux enabled. I'd double check with the team though.

Reply:
interesting; TIL

Reply:
From standup: let's continue to collect this, but let's not analyze it since we think it works with SELinux at this point.